src/components/EtGrid.vue

```diff
@@ -129,7 +130,17 @@ export default {
               value: 1,
             },
           ],
-          fetchRecsName: 'fetchOrdersByStoreId',
+          filterPlaceHolder: 'Type text',
+          fetchRecsByFilterName: 'fetchOrdersByStoreId',
+        }
+      },
+      type: Object,
+    },
+    pagingOptions: {
```

Comment:
prop-"pagingOptions"  &  data-"pagination"  are the same

->> can just create prop-"pagination" with default values